homework/zad9_opencv/pattern_match.cpp

```diff
@@ -0,0 +1,87 @@
+#include "opencv2/imgcodecs.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+#include <iostream>
+#include <mutex>
+#include <future>
+#include <thread>
+
+int main( int argc, char** argv )
+{
+    if (argc < 3)
+    {
+        std::cout << "Not enough parameters" << std::endl;
+        std::cout << "Usage:\n" << argv[0] << " <source_name> <pattern1_name> [<pattern2_name> ...]" << std::endl;
+        return -1;
+    }
+
+    const int patternsCnt = argc - 2;
+
+    cv::Mat imgSource;
+    cv::Mat imgPatterns[patternsCnt];
+
+    // load images
+    imgSource = cv::imread( argv[1], cv::IMREAD_COLOR );
```

Comment:
Inicjalizuj zmienne od razu gdy to możliwe. To powinno być już w linii 20

Reply:
ok